app/presenters/user_presenter.rb

```diff
@@ -0,0 +1,11 @@
+class UserPresenter
+  attr_reader :user
+
+  def initialize(user)
+    @user = user
+  end
+
+  def full_name
+    "#{@user.first_name} #{@user.last_name}"
+  end
+end

```

Comment:
This implementation is correct. This is the simplest one. However, there are various other options. Sometimes we use standard Ruby library `SimpleDelegator`
```ruby
class UserPresenter < SimpleDelegator
  def full_name
    "#{@user.first_name} #{@user.last_name}"
  end
end

presenter = UserPresenter.new(user)
print presenter.last_name # => Diego
print presenter.full_name # => Diego Gómez
```
So, again, in Ruby we like shorter notations, and that's why we use such techniques :) 

It could look like a bit of a magician, and in Enterprise we tend to avoid using too much magic. But that's ok to use some standard widely used approaches, that helps to reduce lines of code and get "out-of-the-box" solutions.